src/Libraries/Microsoft.AspNetCore.Telemetry.Middleware/Logging/LoggingOptions.cs

```diff
@@ -67,7 +67,7 @@ public class LoggingOptions
     /// <value>
     /// The default value is <see cref="HttpRouteParameterRedactionMode.Strict"/>.
     /// </value>
-    [Experimental]
+    [Experimental(diagnosticId: "")]
```

Comment:
@geeknoid @joperezr  we now need to provide diagnostic IDs for all experimental attributes. Is it ok to change it like this for now and fix these later?

Reply:
The URL is optional, but it makes the life of a developer much easier. As a simple starting point we can have aka.ms point to a doc in the repo.